internal/clients/client_models.go

```diff
@@ -30,4 +30,9 @@ type InstanceType struct {
 
 	// The size of the memory, in MiB.
 	Memory int64 `json:"memory,omitempty"`
```

Comment:
Consider including unit of measurement in the name, I do this all the time (e.g. Sec for seconds). This is inspired by systemd and I think it is a reasonable thing because so many configurations and inputs don't do this and don't even document things properly.

```suggestion
	MemoryMiB int64 `json:"memory,omitempty"`
```